ui/app/models/pki/role.js

```diff
@@ -98,18 +98,19 @@ export default class PkiRoleModel extends Model {
     label: 'Not valid after',
     detailsLabel: 'Issued certificates expire after',
     subText:
-      'The time after which this certificate will no longer be valid. This can be a TTL (a range of time from now) or a specific date. If no TTL is set, the system uses "default" or the value of max_ttl, whichever is shorter. Alternatively, you can set the not_after date below.',
+      'The time after which this certificate will no longer be valid. This can be a TTL (a range of time from now) or a specific date.',
```

Comment:
Re: Line 79 - Are we still planning to refactor `backend` as a getter?

Reply:
Yes, good idea I should incorporate that change here